rfcs/block-component.md

```diff
@@ -52,6 +52,59 @@ Props would be exposed for:
 
 > **Note** that we would likely _not_ use this component internally to build other baseui components. It's primarily a helper component for customers when building their apps.
 
+### `Block` API
+
+* `children: (props: Props) => React.Node` - Optional
+* `as: string | React.Node` - Optional
+  * Modifies the base element used to render the block
+* `$style: Object` - Optional
+  * Applies provided styles to Styletron
+* `color: String` - Optional
+  * Accepts all themeable color properties (`primary200`, etc.)
+* `font: String` - Optional
+  * Accepts all themeable font properties (`font200`, etc.)
+* `alignContent: 'start' | 'end' | 'center' | 'between' | 'around' | 'stretch'` - Optional
+* `alignItems: 'start' | 'end' | 'center' | 'baseline' | 'stretch'` - Optional
+* `alignSelf: 'auto' | 'start' | 'end' | 'center' | 'baseline' | 'stretch'` - Optional
+* `direction: 'row' | 'column'` - Optional
```

Comment:
Despite it being a couple more keystrokes, I wonder if we should use `flexDirection` to match the css property name as closely as possible (reduces ambiguity and the need to look up Block API to verify)

Reply:
Updated to `flexDirection`